Update README.md to remove warning and propose steps instead

alexchapar opened the pull request:
## What type of PR is this? (check all applicable)

- [ ] ♻️ Refactor
- [ ] 🐛 Bug Fix
- [x] 👷 Optimization
- [x] 📝 Documentation Update
- [ ] 🔖 Release
- [ ] 🚩 Other

## Description
<!-- Please do not leave this blank -->
Remove the warning about git and propose steps to execute in PowerShell instead

## Related Tickets & Documents
<!-- Please use this format link issue numbers: Fixes #123 https://docs.github.com/en/free-pro-team@latest/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword-->

## Added tests?

- [ ] 👍 yes
- [ ] 🙋 no, because I need help
- [x] 🙅 no, because they aren't needed

## Added to documentation?

- [x] 📜 readme.md (modified)
- [ ] 📜 contributing.md
- [ ] 🙅 no documentation needed

## [optional] What gif/emoji best describes this PR or how it makes you feel?
<!-- You can choose the best that suits you (because YOLO!) : https://www.webfx.com/tools/emoji-cheat-sheet/ -->
<!-- You can use the ones proposed : https://gitmoji.dev/ -->
:zap: :wrench: :page_facing_up: :children_crossing: :technologist:

Bibz87-CG reviewed (changes requested):
Having a section that explains how to display available updates for all apps (`scoop status`) would be useful.

alexchapar commented:
> Having a section that explains how to display available updates for all apps (`scoop status`) would be useful.

I added a little explanation and split other steps to help user chose the command to run if it's for specific apps or for all apps (ex.: `scoop update *` VS `scoop update 7zip` )
